Library/Pyramid Magazine/Pyramid 120/Pyramid 120 Advantages.adq

```diff
@@ -5,6 +5,43 @@
 		<type>Physical, Supernatural</type>
 		<levels>1</levels>
 		<points_per_level>10</points_per_level>
+		<modifier version="1" enabled="no">
```

Comment:
These would be better if left out of the base advantage and instead modifiers were put into a standalone file. I think these come from either Basic or Magic, right?

Reply:
While these have the same name as the Magery modifiers found in Basic, they _do not_ have the same value. The modifier values for Quintessence are half that of the same limitations for normal Magery, and are specific to Quintessence.